pkgs/development/python-modules/tiktoken/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, rustPlatform
+, setuptools-rust
+, libiconv
+, requests
+, regex
+, blobfile
+}:
+let
+  pname = "tiktoken";
+  version = "0.2.0";
```

```suggestion
  version = "0.3.3";
```
Is there a reason you chose an older version?

---

originally I wanted to keep the one that chatgpt-retrieval-plugin was using, but it's working with the most recent version as well so all good.